exonum-java-binding/core/rust/ejb-app/TUTORIAL.md

```diff
@@ -4,7 +4,8 @@ This document describes how to configure and run an Exonum node with a Java serv
 ## Prerequisites
 
 Build an application following the instructions in [“How to Build”][how-to-build] section
-of the Contribution Guide.
+of the Contribution Guide. Unpack the zip archive from `exonum-java-binding/core/target` directory to some known 
```

Comment:
please add the command example, like `unzip exonum-java-binding/core/target/app.zip -d destination_folder`

Reply:
What if user does not have `unzip` honestly I don't think it's necessary